proposals/README.md

```diff
@@ -3,3 +3,4 @@
 ## Proposals
 
 * [Two-Phase Directory-Based Discovery](directory.md)
+* [Vorto](vorto/vorto.md)
```

Comment:
Please add a note that this is informative and would only be included as an example application in the final REC (for example "How to use extension vocabularies for semantic search"